app/presenters/idv/in_person/ready_to_verify_presenter.rb

```diff
@@ -27,10 +27,8 @@ def formatted_enrollment_code
       end
 
       def selected_location_hours(prefix)
-        selected_location_details['hours'].each do |hours_candidate|
-          hours = hours_candidate["#{prefix}Hours"]
-          return localized_hours(hours) if hours
-        end
+        hours = selected_location_details["#{prefix}Hours"]
+        return localized_hours(hours) if hours
```

Comment:
I'm getting super mixed up about the casing.

From this struct it looks like we're passing snake_case hours properties: https://github.com/18F/identity-idp/pull/6624/files#diff-26bde877200f6e82492adec11624842642dea1ba6ff7bc83e40edd3f98481464R5-R16

So how does the Ruby have a camelCase one here?

Reply:
@zachmargolis since we're trying to get #6585 merged which relies on the same data structure, I'm going to defer your suggested case and naming changes to a new ticket, LG-7025.

Reply:
I decided to address this in this PR and did so in 34a7cc50594937453792c6791e0b108e5575c504